### PR TITLE
Add support for one-line public keys

### PR DESCRIPTION
**Pull request by RonnyLV**

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -184,7 +184,7 @@ export async function getApp(
   await app.register(fastifyJWT, {
     secret: configOverrides.jwtKeys ?? {
       private: '-', // Private key blank, as this service won't create JWT tokens, only verify them
-      public: appConfig.jwtPublicKey,
+      public: appConfig.jwtPublicKey.replaceAll('||', '\n'),
```
Comment (kibertoad) [inline src/app.ts:187]: can you add a test for this? AFAIR ConfigScope has refresh method, so you can adjust process.env, refresh configscope and resolve new copy of a config

Comment (kibertoad) [reply src/app.ts:187]: oh, this is in app
maybe we should move this logic to config.ts?

Comment (RonnyLV) [reply src/app.ts:187]: Maybe config.ts would be even a better place

Comment (RonnyLV) [reply src/app.ts]: Rewrote a bit

File: src/infrastructure/config.ts [hunk 2]
```diff
@@ -216,3 +216,7 @@ export function isTest() {
 export function isProduction() {
   return configScope.isProduction()
 }
+
+export function decodeJwtConfig(jwtPublicKey: string) {
```
Comment (dariacm) [inline src/infrastructure/config.ts:220]: 🟢 I don't think it needs to be exported

**Review by dariacm — APPROVED** [review]

File: src/infrastructure/config.ts (hunk 2 above)
Comment (kibertoad) [reply src/infrastructure/config.ts:220]: probably would be much harder to test otherwise

File: src/infrastructure/config.spec.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+})
+
+test('returns an empty string if input is an empty string', () => {
+  expect(decodeJwtConfig('')).toEqual('')
+})
+
+test('throws an error if input is not a string', () => {
+  // @ts-ignore
+  expect(() => decodeJwtConfig(null)).toThrow(TypeError)
+  // @ts-ignore
```
Comment (kibertoad) [inline src/infrastructure/config.spec.ts:22]: ts-expect-error is preferable

**Review by kibertoad — APPROVED** [review]

File: src/infrastructure/config.ts (hunk 2 above)
Comment (dariacm) [reply src/infrastructure/config.ts:220]: Good point, nevermind 😄

File: src/infrastructure/config.spec.ts (hunk 3 above)
Comment (RonnyLV) [reply src/infrastructure/config.spec.ts]: Changed

**Pull request merged**